contracts-periphery/src/UniswapWithdrawHook.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+pragma abicoder v2;
```

Comment:
As mentioned above, this is not needed

Reply:
Now removed https://github.com/ScopeLift/umbra-protocol/pull/341/commits/75faac7fcc1bbfed25b90ee04633145ddcd72693